homeassistant/components/foobot/sensor.py

```diff
@@ -45,11 +43,6 @@
 ATTR_FOOBOT_INDEX = "index"
 
 SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
-    SensorEntityDescription(
```

Comment:
I don't think it's useful to add the entity description, and then filtering out when building the entities with `if description.key != "time"`